_posts/2022-05-19-quinoa-modern-ui-with-no-hassles.adoc

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: 'Quinoa - A Quarkus extension to create Modern UI with no hassles'
+date: 2022-05-19
+tags: quinoa web ui react angular vue lit spa nodejs frontend extension
```

Comment:
please don't add that many tags and try use existing ones.

too many unique ones makes the right side tag menu overcrowded.

Reply:
Maybe the problem is with the menu :) but ok...

Reply:
sure but until then "overloading" tags is problematic.